src/components/Input.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const Input = (props) => {
+
+    return (
+            <fieldset>
+                <label htmlFor={props.id}>{props.label}</label>
+                <input
+                    id={props.id}
+                    onChange={(event) => props.onChange({[props.id]: event.target.value})}
+                    name={props.id}
+                    type={props.type}
+                    value={props.inputVal}
```

Comment:
Should this not be props.value? Since in your prop types below it is defined as value